win32ss/user/ntuser/ghost.c

```diff
@@ -128,6 +219,109 @@ HWND FASTCALL UserHungWindowFromGhostWindow(HWND hwndGhost)
     return IntHungWindowFromGhostWindow(pGhostWnd);
 }
 
+static BOOL IntHaveToQuitGhosting(void)
+{
+    return !!NtUserFindWindowEx(NULL, NULL, &GhostClass, NULL, 0);
```

Comment:
NtUserFindWindowEx acquires the user lock. This is unacceptable for an Int* function

Reply:
Replaced by IntFindWindow.